Tests/ConsoleTest/ConsoleTest.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
```

Comment:
ConsoleTest.Cosmos should be removed, as well as this item group.

Reply:
It is commented on TestKernelSet, I think it could be useful in the future too.

Reply:
Are you using the new project system? Cosmos projects no longer exist, now they're normal projects, and test runner projects shouldn't use the new project system at least for now, because it would require some changes to the AppVeyor configuration.

Reply:
No this work started before you merged the new project system. Do you think it will be better if I do update? How to do this?

Reply:
In this case, simply remove the Cosmos project and the item group, we shouldn't use the new project system for test kernels, at least for now. I'm thinking about doing it in the future, but it would require changes in the AppVeyor configuration and the test runner too.